src/services/merge-checker.ts

```diff
@@ -16,8 +16,15 @@ import { Cache } from './cache'
 
 const logger = require('../services/logger')
 
-const MERGE_BLOCK_NUMBER = 15537394
-const TERMINAL_TOTAL_DIFFICULTY = BigInt('58750003716598352816469')
+const MERGE_BLOCK_NUMBER = {
+  ethereum: 15537394,
+  gnosis: 15537394, // Needs to be updated to the proper block height
+}
+
+const TERMINAL_TOTAL_DIFFICULTY = {
+  ethereum: BigInt('58750003716598352816469'),
+  gnosis: BigInt('8626000000000000000000058750000000000000000000'),
```

Comment:
This will have to be changed to whatever value the Total Difficulty is on the first block after it crosses over the 8626000000000000000000058750000000000000000000 value.

When this happened with the Eth merge, it meant us sitting up and watching for this value: 58750003716598352816469, as the Eth cutoff value was 58750000000000000000000.